### PR TITLE
[zh] Sync access-api-from-pod.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Sync access-api-from-pod.md with en page.

### CODE REVIEW

File: content/zh-cn/docs/tasks/run-application/access-api-from-pod.md [hunk 1]
```diff
@@ -80,16 +80,30 @@ securely with the API server.
...
+Service named `kubernetes` in the `default` namespace so that pods may reference
+`kubernetes.default.svc` as a DNS name for the local API server.
 -->
 #### 直接访问 REST API   {#directly-accessing-the-rest-api}
 
-在运行在 Pod 中时，可以通过 `default` 命名空间中的名为 `kubernetes` 的服务访问
-Kubernetes API 服务器。也就是说，Pod 可以使用 `kubernetes.default.svc` 主机名
-来查询 API 服务器。官方客户端库自动完成这个工作。
+当运行在 Pod 中时，你的容器可以通过获取 `KUBERNETES_SERVICE_HOST` 和 `KUBERNETES_SERVICE_PORT_HTTPS`
+这两个环境变量，来创建一个可以访问 Kubernetes API 服务器的 HTTPS URL。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/access-api-from-pod.md:92]:
```suggestion
这两个环境变量，来生成一个可以访问 Kubernetes API 服务器的 HTTPS URL。
```

File: content/zh-cn/docs/tasks/run-application/access-api-from-pod.md [hunk 2]
```diff
@@ -80,16 +80,30 @@ securely with the API server.
...
+API 服务器的集群内部地址也被发布到命名空间 `default` 中名为 `kubernetes` 的服务，
+以便 Pod 可以将 `kubernetes.default.svc` 引用为本地 API 服务器的 DNS 名称。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/access-api-from-pod.md:94]:
```suggestion
以便 Pod 可以将 `kubernetes.default.svc` 作為本地 API 服务器的 DNS 名称來引用。
```

File: content/zh-cn/docs/tasks/run-application/access-api-from-pod.md [hunk 3]
```diff
@@ -80,16 +80,30 @@ securely with the API server.
...
+
+{{< note >}}
+<!--
+Kubernetes does not guarantee that the API server has a valid certificate for
+the hostname  `kubernetes.default.svc`;
+however, the control plane **is** expected to present a valid certificate for the
+hostname or IP address that `$KUBERNETES_SERVICE_HOST` represents.
+-->
+Kubernetes 不保证 API 服务器具有主机名 `kubernetes.default.svc` 的有效证书；
+然而，控制平面**被期望**为 `$KUBERNETES_SERVICE_HOST` 代表的主机名或
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/access-api-from-pod.md:104]:
```suggestion
然而，控制平面应该为 `$KUBERNETES_SERVICE_HOST` 所代表的主机名或
```

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Comment by tengqm** [comment]
@Zhuzhenghao Please check the merge conflicts. Maybe this PR duplicates another one?

**Comment by Zhuzhenghao** [comment]
@windsonsea Hi, I have rebased upstream/main.

File: content/zh-cn/docs/tasks/run-application/access-api-from-pod.md [hunk 4]
```diff
@@ -80,6 +80,11 @@ securely with the API server.
 <!--
 ### Directly accessing the REST API
 
+While running in a Pod, your container can create an HTTPS URL for the Kubernetes API
+server by fetching the `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT_HTTPS`
+environment variables. The API server's in-cluster address is also published to a
+Service named `kubernetes` in the `default` namespace so that pods may reference
+`kubernetes.default.svc` as a DNS name for the local API server.
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/run-application/access-api-from-pod.md:87]: Line 83-87 is duplicated with line 88-92 now.

**Comment by Zhuzhenghao** [comment]
Closed due to duplicatint to #38990

**Pull request closed without merging**